ThirdParty/Bolts/BoltsInClasspath/README.md.txt

```diff
@@ -0,0 +1,73 @@
+The sample uses weather.com free subscription rest API to demonstrate the Bolts usage. Please register for free and use the token generated from Weather.com and replace it 
+with apiKey placeholder in the Constants.m file
+
+Note: Make sure this folder and Winobjc release folders exist in the same directory. otherwise it may not run.
+
+In the BoltsInClassPath sample app to make this app run on WINOBJC I have found two issues in the app build phase.
+ 
+1) When you start building app you may encounter
+ Bolts.lib (BFExecutor_F456FB2F.obj) : error LNK 2019: unresolved external symbol _pthread_get_stackaddr_np referenced in function remaining_stack_size
+ Bolts.lib (BFExecutor_F456FB2F.obj) : error LNK 2019: unresolved external symbol _pthread_get_stacksize_np referenced in function remaining_stack_size
+
+This happens because I see a different implementations for pThread on Winobjc. so my temp workaround is
+
+go to your Bolts dependency in the solution and navigate to Bolts/Bolts/Common/BFExecutor.m file. change the below method implementation like this.
+
```

Comment:
As I mentioned in the Discussion, I recommend the use of git submodule instead of dragging along the complete framework.

In this case, I suggest that you provide directions to add the submodule, then update the Bolts framework as you suggest below.